README.md

```diff
@@ -324,10 +324,19 @@ Table component supporting sorting and filtering.
 | data (required) | Array | The array of data items, each corresponding to a single potential table row | None
 | filter (optional) | Function | Called with data for a single row. Should return `false` if the row should be filtered out, or `true` otherwise. | None
 | fixed (optional) | Boolean | Whether or not table column widths should be fixed (vs fluid). | False
+| initialPage (optional) | Number | The initial page to be displayed initially. | None
 | initialSortState (optional) | `{columnID: String, direction: Table.sortDirection}` | The initial sort state of the table. | None
+| onPageChange (optional) | Function | Callback function for the displayed page change event. | None
```

Comment:
yeah, decided to keep the base table component simple and not bake in any async functionality.
we can achieve implement the load-on-page-scroll pattern by having placeholder cell contents (or loading animation that get replaced once the data is fetched. can add an example to the table demo in a follow-up PR